docs/source/gbd2019_models/risk_exposures/stunting/index.rst

```diff
@@ -0,0 +1,337 @@
++       +-------+-----------------------+--------------------+
+|       | SEV   | < -3 Z score          | 2.5 (1.9 - 3.4)    |
++-------+-------+-----------------------+--------------------+
+| LRI   | MILD  | < -1 to -2 Z score    | 1.2 (1.0 - 1.5)    |
+|       +-------+-----------------------+--------------------+
+|       | MOD   | < -2 to -3 Z score    | 1.4 (1.1 - 1.9)    |
+|       +-------+-----------------------+--------------------+
+|       | SEV   | < -3 Z score          | 2.7 (1.7 - 4.4)    |
++-------+-------+-----------------------+--------------------+
+| MSLS  | MILD  | < -1 to -2 Z score    | 1.1 (0.6 - 1.8)    |
+|       +-------+-----------------------+--------------------+
+|       | MOD   | < -2 to -3 Z score    | 2.0 (1.3 - 3.5)    |
+|       +-------+-----------------------+--------------------+
+|       | SEV   | < -3 Z score          | 3.7 (2.0 - 8.0)    |
++-------+-------+-----------------------+--------------------+
+
++-------+-------+-----------------------+--------------------+
+|2y-4y  | TMREL |  >= -1                | 1                  |            
++-------+-------+-----------------------+--------------------+
+| DD    | MILD  | < -1 to -2 Z score    | 1.2 (1.1 - 1.3)    |
+|       +-------+-----------------------+--------------------+
+|       | MOD   | < -2 to -3 Z score    | 1.4 (1.2 - 1.6)    |        
++       +-------+-----------------------+--------------------+
+|       | SEV   | < -3 Z score          | 2.5 (1.9 - 3.4)    |
++-------+-------+-----------------------+--------------------+
+| LRI   | MILD  | < -1 to -2 Z score    | 1.2 (1.0 - 1.5)    |
+|       +-------+-----------------------+--------------------+
+|       | MOD   | < -2 to -3 Z score    | 1.4 (1.1 - 1.9)    |
+|       +-------+-----------------------+--------------------+
+|       | SEV   | < -3 Z score          | 2.7 (1.8 - 4.5)    |
++-------+-------+-----------------------+--------------------+
+| MSLS  | MILD  | < -1 to -2 Z score    | 1.1 (0.6 - 1.9)    |
+|       +-------+-----------------------+--------------------+
+|       | MOD   | < -2 to -3 Z score    | 2.0 (1.2 - 3.8)    |
+|       +-------+-----------------------+--------------------+
+|       | SEV   | < -3 Z score          | 3.7 (2.0 - 8.0)    |
++-------+-------+-----------------------+--------------------+
+
+
+Vivarium Modeling Strategy
+++++++++++++++++++++++++++
+
+Child stunting in GBD has an **ordered polytomous variable**. It has rei_id 241
+
+We will use a 'propensity exposure model' for child stunting, in which each simulant is initialized with a “propensity” for a stunting category, and the simulant’s stunting category is determined by comparing this propensity to the overall stunting exposure/prevalence in the population. 
+
+.. todo::
+
+  Discuss with engineers if we want to use the propensity model or the invidivual Z-score distributions? Confirm with James how stunting was modelled in BEP.
```

Comment:
I suggest that we use an approach that allows us to include a relative risk for extreme stunting if GBD modelers can provide us with that value.  I think we also need an approach where we can correlate stunting and wasting (perhaps through the X Factor).

Reply:
hmm I dont think they have RRs for extreme stunting. I can double check.